hotel-api-persistence/src/main/java/com/fponce/hotelapp/persistence/HotelRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.fponce.hotelapp.persistence;
+
+import java.sql.SQLException;
+import java.util.UUID;
+
+public interface HotelRepository {
+
+    void createHotel(UUID id, String name, int category) throws SQLException;
```

Comment:
Wouldn't it be useful adding proper javaDoc for this method?

Reply:
Yes, I added proper javadoc. I found this resources useful, not sure if there is a better approach.
https://dev.liferay.com/participate/javadoc-guidelines
https://zeroturnaround.com/rebellabs/reasons-tips-and-tricks-for-better-java-documentation/